components/Projects/FiltersBar/FilterBtnsGroup.tsx

```diff
@@ -9,60 +9,68 @@ export interface ProjectFilter {
 }
 
 // TODO: Move mock data to external folder
-export const filters: ProjectFilter[] = [
-  {
-    name: 'Java',
-    isActive: false,
-  },
-  {
-    name: 'Python',
-    isActive: false,
-  },
-  {
-    name: 'CSS',
-    isActive: false,
-  },
-  {
-    name: 'Spring',
-    isActive: false,
-  },
-  {
-    name: 'Javascript',
-    isActive: true,
-  },
-  {
-    name: 'Angular',
-    isActive: false,
-  },
-  {
-    name: 'Vue',
-    isActive: false,
-  },
-  {
-    name: 'Next.js',
-    isActive: true,
-  },
-  {
-    name: 'Node.js',
-    isActive: false,
-  },
-  {
-    name: 'C#',
-    isActive: false,
-  },
-  {
-    name: 'C++',
-    isActive: false,
-  },
-  {
-    name: 'React',
-    isActive: true,
-  },
-];
+// export const filters: ProjectFilter[] = [
+//   {
+//     name: 'Java',
```

Comment:
Did you put this in a comment because it interrupted the `FilterBtnsGroup` props?

Reply:
This was a stub prior to implementation.
I removed the commented code out in last commit.